app/src/main/java/com/github/bsideup/liiklus/config/RecordPostProcessorChain.java

```diff
@@ -0,0 +1,19 @@
+package com.github.bsideup.liiklus.config;
+
+import com.github.bsideup.liiklus.records.RecordPostProcessor;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import java.util.Collection;
+
+@RequiredArgsConstructor
+@FieldDefaults(makeFinal = true)
+public class RecordPostProcessorChain {
+
+    Collection<RecordPostProcessor> processors;
```

Comment:
is better to store it as List explicitly indicating that collection is sorted?

Reply:
no, did it intentionally, see TODO in `getAll`. Chains should sort them explicitly (will be added later)